doc/src/manual/environment-variables.md

```diff
@@ -406,6 +406,14 @@ should have at the terminal.
 The formatting `Base.answer_color()` (default: normal, `"\033[0m"`) that output
 should have at the terminal.
 
+### `NO_COLOR`
+
+If set to anything besides `""`, then colored text will be disabled on the REPL. Can be overridden with the `--color=yes` flag. This environmental variable is [commonly recognized by command-line applications](https://no-color.org/).
+
+### `FORCE_COLOR`
+
+If set to anything besides `""`, then colored text will be forced on the REPL. Can *not* be overridden with the `--color` flag. This environmental variable is [commonly recognized by command-line applications](https://force-color.org/).
```

Comment:
I don't understand why `FORCE_COLOR` has precedence over `--color`. That seems unusual, and somewhat problematic. I read the manifesto for FORCE_COLOR and it takes about setting the *default* to be "color on", it doesn't say that it forbids higher priority overrides.

Reply:
I'll double check but I believe I did this because I couldn't tell the different from the user explicitly setting `--color=auto` and the implicit `--color=auto`.

Reply:
Okay in reviewing https://force-color.org/ again I see the example implementation has the `force_color` setting being applied *after* the CLI options for the application. I agree with you that the CLI options should take precedence so I'll try to refactor things to respect that.